### DISCUSSION TITLE
My CSS is not working

### DISCUSSION THREAD

**Discussion by ItTRA-Monica** [community/community#56458] [Programming Help]
### Select Topic Area

Question

### Body

Hello eveyone,

I am new to github and new in doing this. And I've been watching on how to set-up pages. 
Now my problem is I am trying to move my HTML and CSS here in github. My HTML is working, but apparently CSS is not being applied. 

I think I did something wrong but I cannot figure it out. Can someone help me ?

https://ittra-monica.github.io/ItServicesVersion1/

Thank you in advance for anyone can help me.

**Comment by radar-blip** [comment]
You need to change the link to your style.css (in your index.html)
from this:
`<link rel="stylesheet" href="css/test.css">`

to this:
`<link rel="stylesheet" href="style.css">`

**Comment by jayantur13** [comment]
I think there is mistake in path. If you have in root css(folder)/test.css(css file) then path may look like this 
`<link rel="stylesheet" href="./css/test.css ">`

If you have test.css in root then
`<link rel="stylesheet" href="test.css"> `

**Comment by ItTRA-Monica** [comment]
Thank you so much for the help. Really appreciate it

**Comment by ElyasShamal** [comment]
I think there is a mistake in the path  check your CSS link in HTML

**Comment by Venkatesh-KCET** [comment]
I have fixed the problem and sent a request for you to review the code and accept it. 

There was a small mistake where you uploaded all the files in the main folder instead of placing the images and CSS files in their respective subfolders. 

I have corrected this issue. 

Please consider using Git or GitHub Desktop for future uploads. A

lso, please remember to follow me and mark my answer as correct.
